core-httpclient-impl/src/main/java/com/optimizely/ab/config/HttpProjectConfigManager.java

```diff
@@ -367,7 +366,7 @@ public HttpProjectConfigManager build(boolean defer) {
                 blockingTimeoutUnit,
                 notificationCenter,
                 sdkKey);
-
+            httpProjectManager.setSdkKey(sdkKey);
```

Comment:
Isn't this `seSdkKey` is redundant? I see `sdkKey' parameter to the constructor (line 368).

Reply:
This will ensure that to trigger notification user provided sdkKey will always be prioritized, instead of what is available in ProjectConfig. This allows registry to always trigger notification even if the SDKKey is changed in the Projectconfig.

Reply:
@mnoman09 If we need this extra description for proper initializing sdkKey, this may be a source of issues. If you have to call setSdkKey() for proper overriding, can we remove sdkKey from constructor?

Reply:
I already removed it from the HttpProjectConfigManager constructor if that is what you are asking?

Reply:
@mnoman09 looks good!